test-utils/runtime/src/genesismap.rs

```diff
@@ -81,11 +81,21 @@ impl GenesisConfig {
 		// Assimilate the system genesis config.
 		let mut storage =
 			Storage { top: map, children_default: self.extra_storage.children_default.clone() };
-		<system::GenesisConfig as GenesisBuild<Runtime>>::assimilate_storage(
-			&system::GenesisConfig { authorities: self.authorities.clone() },
+
+		<substrate_test_pallet::GenesisConfig as GenesisBuild<Runtime>>::assimilate_storage(
```

Comment:
We should use the `crate::GenesisConfig` here and then only give the possibility to override keys by adding some kind of `HashMap<Vec<u8>, Vec<u8>>` and then add this to the storage after putting the genesisconfig into the storage.

Reply:
Done in: https://github.com/paritytech/substrate/pull/13737/commits/88683c877fb757946fd7fbfd69ebb923b1335d16 and https://github.com/paritytech/substrate/pull/13737/commits/b71a5b3ccc0ab27c4b38cce5c34a16a7ff9f7aee